src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicSingleBlackHole.hpp

```diff
@@ -110,11 +151,21 @@ struct EvolutionMetavars
 
   using typename gh_base::step_actions;
 
+  // Add ElementInitInterpPoints to initialization_actions.
+  using helper_init =
+      tmpl::push_back<tmpl::pop_back<initialization_actions>,
+                      intrp::Actions::ElementInitInterpPoints<
+                          intrp::Tags::InterpPointInfo<EvolutionMetavars>>,
+                      tmpl::back<initialization_actions>>;
+
+  // the dg element array needs to be re-declared to capture the new type
+  // aliases for the action lists.
   using gh_dg_element_array = DgElementArray<
       EvolutionMetavars,
       tmpl::flatten<tmpl::list<
-          Parallel::PhaseActions<Parallel::Phase::Initialization,
-                                 initialization_actions>,
+          Parallel::PhaseActions<
+              Parallel::Phase::Initialization,
+              tmpl::list<initialization_actions, helper_init>>,
```

Comment:
this seems to add init actions twice. with the change I commented above you can just drop this.